src/main/java/org/opensearch/security/action/configupdate/ConfigUpdateNodeResponse.java

```diff
@@ -78,4 +80,14 @@ public void writeTo(StreamOutput out) throws IOException {
     public String toString() {
         return "ConfigUpdateNodeResponse [updatedConfigTypes=" + Arrays.toString(updatedConfigTypes) + ", message=" + message + "]";
     }
+
+    @Override
+    public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
```

Comment:
Why is this needed?

Reply:
For exposing Requests and Responses on the REST layer, you need to implement `ToXContent` to be able to serialize to and from JSON.